src/OrchardCore/Orchard.Logging.NLog/ApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Logging;
+using NLog.Extensions.Logging;
+using NLog.LayoutRenderers;
+using NLog.Web;
+
+namespace Orchard.Logging
+{
+    public static class ApplicationBuilderExtensions
+    {
+        public static IApplicationBuilder AddNLogWeb(this IApplicationBuilder app, ILoggerFactory loggerFactory)
+        {
+            LayoutRenderer.Register<TenantLayoutRenderer>(TenantLayoutRenderer.LayoutRendererName);
```

Comment:
I would have let this code in the application and just put the class in the project.

The same way you do `loggerFactory.AddConsole()` and don't call an extension method which is in the console package.